torch/csrc/jit/passes/quantization.cpp

```diff
@@ -490,8 +583,7 @@ void QuantizeHelper::quantizeTensor(Value* v, bool insert_after) {
 
 c10::optional<script::Module> QuantizeHelper::findChildModuleToQuantize(
     Value* v) {
-  if (v->node()->kind() == prim::CallMethod &&
-      v->node()->s(attr::name) == "forward") {
+  if (v->node()->kind() == prim::CallMethod) {
```

Comment:
why is check for "forward" removed? Is it because this will be called recursively and child may not have "forward" ?

Reply:
yes, we are quantizing all the method that is called